runbotics/runbotics-orchestrator-ui/src/src-landing/components/BlogHeadImage/BlogHeadImage.module.scss

```diff
@@ -1,25 +1,31 @@
 @import '#src-landing/scss/main.scss';
 
-.background{
-    object-fit: 'cover'
-}
+$BACKGROUND-MAX-WIDTH:  1630px;

```

Comment:
use `MAX_SECTION_CONTENT_WIDTH` + add padding same as on blog content. tbh it should be a single wrapper for content and banner image